data/categories.txt

```diff
@@ -8816,6 +8848,7 @@ nb:^Billigbutikk
 nl:^Bazaar
 pl:^Sklep z różnościami
 pt:^Loja dos 300
+pt-BR:^1,99|Loja de variedades
```

Comment:
nit: "1,99" will not work as you expect in the current version (nor in the near future)
We'll most likely have to remove it.

Reply:
Because it starts with a number? If it gets updated later to "Loja de 1,99", will it be valid then?

Reply:
Yes. And also because we change commas and some other symbols to spaces during simplification.
"Loja de 1,99" is better but still may have issues.

Reply:
Will a dot be better than a comma? "Loja de 1.99", for example.

Reply:
The full list is here: https://github.com/mapsme/omim/blob/master/indexer/search_delimiters.cpp

The rule of thumb is that most non-alphanumeric tokens are transformed to spaces.
Let's keep it as is for now.

Reply:
Right, thanks!